source/plugins/ruby/out_mdm.rb

```diff
@@ -408,5 +422,14 @@ def isWindows()
       end
       return isWindows
     end
+
+    def valid_url?(string)
+      uri = URI.parse(string)
+      %w( http https ).include?(uri.scheme)
```

Comment:
what this **%w( http https ).include?(uri.scheme)** do?

Reply:
It's used to verify if this uri starts with http or https. Considering this endpoint is setup by ArcA admin, it not necessary here.